charts/openyurt/templates/yurt-manager-auto-generated.yaml

```diff
@@ -729,7 +729,7 @@ webhooks:
   clientConfig:
     service:
       name: webhook-service
-      namespace: kube-system
+      namespace: {{ .Release.Namespace | quote }}
```

Comment:
`charts/openyurt/templates/yurt-manager-auto-generated.yaml` is generated automatically, it is not need to modify the contents in this file.